style.css

```diff
@@ -159,10 +159,14 @@ body {
   padding: 0 15px 0 15px;
 }
 .login-button {
-  margin-top: 8px;
+  margin-top: 15px;
```

Comment:
It looks like the log-in button is closer to the white area.  Maybe take a little bit of the margin-top off, or use flexbox to center it.  
![image](https://user-images.githubusercontent.com/42394859/118074464-f48a8480-b37b-11eb-88da-b5ab14eef434.png)

Reply:
Definately, Thanks